layout/layout_test.cpp

```diff
@@ -1139,5 +1139,40 @@ int main() {
         expect_eq(layout::create_layout(style, 0), std::nullopt);
     });
 
+    etest::test("xpath", [] {
+        dom::Node html_node = dom::Element{"html"s};
+        dom::Node div_node = dom::Element{"div"s};
+        dom::Node p_node = dom::Element{"p"s};
+        dom::Node text_node = dom::Text{"hello!"s};
+        style::StyledNode styled_node{
+                .node = html_node,
+                .properties{{css::PropertyId::Display, "block"}},
+                .children{
+                        style::StyledNode{.node = div_node, .properties{{css::PropertyId::Display, "block"}}},
+                        style::StyledNode{.node = text_node},
+                        style::StyledNode{.node = div_node,
+                                .children{
+                                        style::StyledNode{.node = p_node},
+                                        style::StyledNode{.node = text_node},
+                                }},
+                },
+        };
+        auto layout = layout::create_layout(styled_node, 123).value();
+
+        // Verify that we a shady anon-box to deal with in here.
```

Comment:
Wording

Reply:
Whoops. I accidentally a word. :( That's what I get for writing comments. Good catch!